README.md

```
@@ -1,23 +1,56 @@
+  <img src="docs/images/node-screenshot.png" alt="SurrealDB node in n8n">
 </p>
 
-[n8n](https://n8n.io/) is a [fair-code licensed](https://docs.n8n.io/reference/license/) workflow automation platform.
+## Documentation
+
+View the node documentation [here](https://surrealdb.com/docs/integrations/data-management/n8n).
+
+## Learn SurrealDB
+
+- SurrealDB University: https://surrealdb.com/learn/fundamentals
+- Aeon's Surreal Renaissance (Interative book): https://surrealdb.com/learn/book
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)